Doc/library/sqlite3.rst

```diff
@@ -1604,7 +1604,8 @@ Row objects
    It supports iteration, equality testing, :func:`len`,
    and :term:`mapping` access by column name and index.
 
-   Two row objects compare equal if they have equal columns and equal members.
+   Two :class:`!Row` objects compare equal
+   if they have equal columns and equal members.
```

Comment:
```suggestion
   if they have the identical columns and values.
```

* "Values" is, I think, a more specific and appropriate word here, since a member of a named tuple includes both its names and its values, and is a more abstract and less commonly understood concept than the values in each column/of each member.
* At least to me as a native speaker, "equal columns" to me implies more an equal _number_ of columns, since "equal" is generally mostly reserved for numeric quantities, as opposed to _identical_ which seems to more closely match the intended meaning here (that the column names and order are the same).
* Avoid repeating the verb three times—I thought about saying "equal values", but the fact that a different word is used for each seems to me to imply that the equivalence testing is different between columns and values, (i.e. one is identity and one is equality), when AFAIK it is not. Most other words I tried were not strong or clear enough.

Reply:
+1, but I find the added _the_ off-kilter. Why not this?

```suggestion
   if they have identical columns and values.
```

Reply:
I'll get to reviewing and responding to the rest of it shortly, but yeah the definite article was an editing typo here, sorry :+1: